Added link to website bindings list in README.md (#562)
gfx-rs/wgpu-native: README.md

# Bindings

> [!NOTE]

> > *An additional, better-sorted and -updated list is available on the [wgpu website](https://wgpu.rs/#bindings).*

Reviewer: ```suggestion
> *This list is also available on the [wgpu website](https://wgpu.rs/#bindings).*
```

I don't really like the superlatives as I think it will still be useful to have both lists
